packages/bsky/src/auth-verifier.ts

```diff
@@ -234,7 +234,15 @@ export class AuthVerifier {
         )
       })
 
-    const { sub, aud, scope } = res.payload
+    const { sub, aud, scope, cnf } = res.payload
+    if (typeof cnf !== 'undefined') {
+      // Proof-of-Possession (PoP) tokens are not allowed here
+      // https://www.rfc-editor.org/rfc/rfc7800.html
+      throw new AuthRequiredError(
+        'Malformed token: DPoP not supported',
```

Comment:
In the PDS we just give `Malformed token`, but here I think it's probably wise to be explicit and say "DPoP isn't supported", since getting here in the first place would be already a developer error (using `Bearer <token>` with a DPoP bound access token.